yt_dlp/extractor/arte.py

```diff
@@ -165,6 +184,7 @@ def _real_extract(self, url):
             if 'HLS' in stream['protocol']:
                 fmts, subs = self._extract_m3u8_formats_and_subtitles(
                     stream['url'], video_id=video_id, ext='mp4', m3u8_id=stream_version_code, fatal=False)
+                subs = self._contvert_accessible_subs_locale(subs)
```

Comment:
I think this line should be put below, where the `self._merge_subtitles` is, or even inlined there. That would have them be thematically linked